templates/ingress.yaml

```diff
@@ -409,3 +409,38 @@ spec:
   {{- end }}
 {{- end }}
 ---
+{{- if and (merge .Values dict | dig "coderd" "serviceNext" true) (merge .Values dict | dig "ingress" "enable" true) }}
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: coderd-ingress
+  namespace: {{ .Release.Namespace | quote }}
+  annotations:
+    nginx.ingress.kubernetes.io/proxy-body-size: "0"
```

Comment:
I think this can be put in the default values file, then we can make all of the annotations configurable, similar to how we do it with securityContext: https://github.com/cdr/enterprise-helm/blob/e158a751b9a1c3198adff2c08b4a5f50c1e33fdf/templates/coderd.yaml#L36-L42

right now there's no way to clear the `nginx.ingress.kubernetes.io/proxy-body-size` annotation here

FWIW, I think this resolves: https://app.clubhouse.io/coder/story/14644/route-traffic-to-coderd-using-ingress -- thank you!